packages/itwinui-css/src/menu/mixins.scss

```diff
@@ -180,7 +180,7 @@ $iui-active-outline: 1px solid var(--iui-color-border-accent);
 @mixin iui-menu-icon {
   flex: 0 0 auto;
 
-  &,
+  &.iui-skeleton,
```

Comment:
this change is breaking some tests. why is it needed?

Reply:
Not needed anymore, I added it to fix skeleton icon with iui-list-item css.